src/common/vfs.cpp

```diff
@@ -166,9 +166,9 @@ bool OCC::isVfsPluginAvailable(Vfs::Mode mode)
 
 Vfs::Mode OCC::bestAvailableVfsMode()
 {
-    if (isVfsPluginAvailable(Vfs::WindowsCfApi)) {
-        return Vfs::WindowsCfApi;
-    } else if (isVfsPluginAvailable(Vfs::WithSuffix)) {
+//    if (isVfsPluginAvailable(Vfs::WindowsCfApi)) {
+//        return Vfs::WindowsCfApi;
```

Comment:
Please do not just comment out code, please provide context in a comment, why this code is disabled and under what circumstances it can be reenabled again.